jekyll/_cci2/release/set-up-a-release-environment.adoc

```diff
@@ -39,14 +39,13 @@ CircleCI supports Kubernetes, Helm, and Argo Rollouts. The following versions ha
 {% include snippets/release/supported-versions.adoc %}
 
 [#create-release-environment]
-== 1. Create your release environment and token
+== 1. Create your release environment
 
 . In the CircleCI web app, select **Releases** in the sidebar.
-. Select btn:[Add Release Environment].
-. Enter a name for your release environment and select btn:[Create Environment].
-. Select the environment you just created.
-. Select btn:[Create New Token] and then btn:[Create Token]
-. The new token will be displayed. Be sure to copy and save this somewhere for use in the next step. Then select btn:[Done]
+. Select btn:[Create your first Environment Integration].
+. Enter a name for your release environment and click btn:[Next: Release Agent Setup].
+. The onboarding wizard will then guide you through the next steps.
+. After completing all the steps in the wizard you would have successfully created a release environment and installed the release agent into your Kubernetes cluster and configured your Kubernetes components
 
 [#add-release-agent]
```

Comment:
question: Do we need to keep rest of the documentation? Or would it be better to remove the rest since it will be covered by Onboarding wizard anyways?

Reply:
What happens when you want to create a second release env?

Reply:
Google Meet call discussion notes

- We will prioritise updating the public docs to remove incorrect and invalid information (ie The token creation step post environment creation is now not possible due to introduction of the wizard)
- We need to update usage of the term `Release Environment` in the docs to `Environment Integration`


@rosieyohannan I just realised something that I missed during the call. The user can still create new tokens manually from the environment settings page instead of copying it from the wizard. I have updated the PR to reflect this. (and also added info on what happens when you create a second release env)